org.freecad.FreeCAD.yaml

```diff
@@ -0,0 +1,802 @@
+app-id: org.freecad.FreeCAD
+app-id: org.freecad.FreeCAD
+default-branch: beta
+runtime: org.kde.Platform
+runtime-version: '6.7'
+sdk: org.kde.Sdk
+command: FreeCAD
+desktop-file-name-prefix: "(beta) "
+rename-mime-icons:
+  - application-x-extension-fcstd
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=network
+  - --filesystem=host # needed to make file saving work
+  - --filesystem=xdg-run/gvfs # make GnomeVFS accessible
+  - --filesystem=/run/spnav.sock:ro # allow access to spacenavd
+  - --talk-name=org.freedesktop.Flatpak # to launch openscad
+  - --env=TMPDIR=/var/tmp
+  - --env=SPNAV_SOCKET=/run/spnav.sock
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.llvm15
+cleanup:
+  - /include
+  - /share/aclocal
+  - /share/cmake
+  - /lib/cmake
+  - /share/man
+  - '*.a'
+  - '*.la'
+build-options:
+  append-path: /usr/lib/sdk/llvm15/bin
+  append-ld-library-path: /usr/lib/sdk/llvm15/lib
+modules:
+
+  - shared-modules/glu/glu-9.json
+
+  - name: pyside
```

Comment:
we have a pyside baseapp now. can that be used ?

Reply:
do they disable py_limited_api in that build?

Reply:
Don't think so, the default is to build without limited API.